gen3-client/commonUtils/commonUtils.go

```diff
@@ -15,6 +15,15 @@ import (
 	pb "gopkg.in/cheggaaa/pb.v1"
 )
 
+// ShepherdEndpoint is the endpoint postfix for SHEPHERD / the Object Management API
+const ShepherdEndpoint = "/mds"
+
+// ShepherdVersionEndpoint is the endpoint used to check what version of Shepherd a commons has deployed
+const ShepherdVersionEndpoint = "/mds/version"
+
+// MinAcceptableShepherdVersion is the minimum version of Shepherd that will work with this version of the client.
+const MinAcceptableShepherdVersion = "2.0.0"
```

Comment:
maybe should make this configurable, modify the `configure` cmd to take an optional input from user and save it with the profile